### PR TITLE
[FIX][8.0] demo data should not be regular data

### PR DESCRIPTION
**Pull request by thomaspaulb**

### CODE REVIEW

File: l10n_nl_xaf_auditfile_export/__openerp__.py [hunk 1]
```diff
@@ -13,6 +13,9 @@
         'base',
         'account',
     ],
+    "demo": [
+        "demo/res_partner.xml"
+    ],
     "data": [
         "demo/res_partner.xml",
```
Comment (astirpe) [inline l10n_nl_xaf_auditfile_export/__openerp__.py]: I think you forgot to remove this line inside "data" ?

**Review by astirpe — CHANGES_REQUESTED** [review]
Could you update the version number also?

File: l10n_nl_xaf_auditfile_export/__openerp__.py (hunk 1 above)
Comment (thomaspaulb) [reply l10n_nl_xaf_auditfile_export/__openerp__.py]: Very true, fixed now.

**Review by astirpe — APPROVED** [review]
Runbot failure doesn't seem to be related with this PR

**Review by tarteo — APPROVED** [review]

**Pull request merged**